docs/video/01-build/04-Develop-Initializing-Video-API.mdx

```diff
@@ -59,15 +54,17 @@ Here below is how to initializing Video API:
 | -            | `options.config.audio`   | `boolean`     | -       | pass `true`to enable audio on start, else pass `false`.                                                                                                                     |
 | -            | `options.media`          | `MediaStream` | -       | Local stream. For backend use. Defaults to `null`.                                                                                                                          |
 
+> Note: Parameters _`in this style`_ are mandatory.
+
 Now let's go over each property individually and understand how to set them.
 
 ### Understanding data variable
 
 - The `data` variable is an object that is used to store all of the video call-related data.
-- It is of type `PushAPI.VideoCallData` and should be initialized with `initVideoCallData`.
+- It is of type `TYPES.VIDEO.DATA` and should be initialized with `CONSTANTS.VIDEO.INITIAL_DATA`.
 - For React-based projects, this would basically be a React state, and for vanilla JS projects, it will be a simple JS variable.
 
-The type `PushAPI.VideoCallData` is described below:
+The type `TYPES.VIDEO.DATA` is described below:
```

Comment:
Maybe have an overarching description so it should be Understanding video lifecycle, setData, stream and media stream object can come under it. This should be H2 (##)